README.md

```diff
@@ -1,7 +1,7 @@
 # bevy_console
 [![Check](https://github.com/RichoDemus/bevy-console/actions/workflows/build.yaml/badge.svg)](https://github.com/RichoDemus/bevy-console/actions/workflows/build.yaml)
 
-A simple half-life inspired console with support for argument parsing.
+A simple half-life inspired console with support for argument parsing powered by `clap`.
```

Comment:
```suggestion
A simple *Half Life* inspired console with support for argument parsing powered by `clap`.
```

Not your mistake, but while we're here...

Reply:
It's *Half-Life* the dash is in the title, but the words are capitalized.